src/main/java/com/wcc/platform/service/MentorshipService.java

```diff
@@ -27,11 +32,19 @@
 public class MentorshipService {
   private final ObjectMapper objectMapper;
   private final PageRepository repository;
+  private final MentorsRepository mentorsRepository;
+  private final HeroSectionRepository heroSectionRepository;
 
   @Autowired
-  public MentorshipService(final ObjectMapper objectMapper, final PageRepository repository) {
+  public MentorshipService(
+      final ObjectMapper objectMapper,
+      final PageRepository repository,
+      final MentorsRepository mentorsRepository,
+      final HeroSectionRepository heroSectionRepository) {
```

Comment:
I don't think we should be mixing CMS pages with Mentorship object. The creation of the page of mentors is not the same as creation of a single mentor, imagine this as administration and the other is just the CMS, we can talk about that in our next meeting.